REQUIRE

```diff
@@ -6,4 +6,4 @@ Documenter 0.7.1
 NamedArrays 0.5.3
 Plots 0.9.4
 JSON
-
+InfoZIP
```

Comment:
I'd prefer if this dependency was not listed here. For normal use of Mimi one doesn't need this package, really just for testing, which is probably a very small subset of people that will install Mimi. The previous version had this ``Pkg.installed("ZipFile")`` code at the top of ``test_dependencies.jl``, could you just add that back in but modify it to install InfoZIP if needed?